lib/solidus_feeds/generators/google_merchant.rb

```diff
@@ -0,0 +1,93 @@
+# lib/google_merchant.rb
+
+require 'builder'
+
+module SolidusFeeds
+  module Generators
+    # The GoogleMerchant XML feed as described in https://support.google.com/merchants/answer/7052112.
+    class GoogleMerchant
+      attr_accessor :products, :host
+
+      def initialize(products, host:)
+        self.products = products
+        self.host = host
+      end
+
+      def call(io)
+        builder = Builder::XmlMarkup.new(target: io, indent: 0)
+        render_template(builder)
+      end
+
+      private
+
+      def render_template(xml)
+        xml.rss version: "2.0", "xmlns:g" => "http://base.google.com/ns/1.0" do
+          xml.channel do
+            xml.title 'store title'
+            xml.link 'store link'
+            xml.description 'store description'
+            xml.language 'store lang'
+            products.find_each do |product|
+              xml.item do
+                xml.tag! 'g:id', product.id
+                xml.tag! 'g:title', product.name
+                xml.tag! 'g:description', product.description
```

Comment:
If we had methods for these, it would be easier to override them and it would be more consistent. What do you think?